inductiva/fluids/scenarios/coastal_area/coastal_area.py

```diff
@@ -0,0 +1,131 @@
+"""Coastal area scenario."""
+
+from functools import singledispatchmethod
+import math
+import os
+from typing import Optional
+import shutil
+from uuid import UUID
+
+from inductiva.types import Path
+from inductiva.scenarios import Scenario
+from inductiva.simulation import Simulator
+from inductiva.fluids.simulators import SWASH
+from inductiva.utils.templates import (TEMPLATES_PATH,
+                                       replace_params_in_template)
+
+SCENARIO_TEMPLATE_DIR = os.path.join(TEMPLATES_PATH, "coastal_area")
+SWASH_TEMPLATE_SUBDIR = "swash"
+SWASH_CONFIG_TEMPLATE_FILENAME = "input.sws.jinja"
+SWASH_CONFIG_FILENAME = "input.sws"
+
+
+class CoastalArea(Scenario):
+    """Coastal area scenario."""
+
+    valid_simulators = [SWASH]
+
+    def __init__(self, wave_amplitude: float = 2, wave_period: float = 10):
+        """Initializes a `CoastalArea` object.
+
+        Args:
+            wave_amplitude: The amplitude of the wave, in meters.
+            wave_period: The period of the wave, in seconds.
+        """
+        self.wave_amplitude = wave_amplitude
+        self.wave_period = wave_period
+
+    def simulate(
+        self,
+        simulator: Simulator = SWASH(),
```

Comment:
We can simply pass the bathymetry here as a .bot file. In this way, it will be simple for anyone to use. 

This follows even if we add a bathymetry class to manage them, in the end, what we would pass is .bot file.